ansible/files/steps/batch.sh

```diff
@@ -89,13 +90,14 @@ else
     if [ $EXTENSION = "json" ]; then
         # Executes Zeppelin notebook
         source activate zeppelin
-        zeppelin-execute -i ${NOTEBOOK_NAME}
+        zeppelin-execute -i ../${NOTEBOOK_NAME} -o ./note.json
         EXIT_CODE=$?
+        zeppelin-convert -i ../note.json -o ./${FILE_NAME}.md
```

Comment:
Would it be safer to use the original name? Would there ever be a conflict with `note.json`?